### PR TITLE
Sync with official fan translations

### PR DESCRIPTION
**Pull request by Vankog**
status: done
implementation of #2

* Synchronizing the phrases with 
   * the officially listed fan translation PDFs from the CAH website by "Balz" and "Raphael" 
   * and the official US PDF.
* Adding comments: 
   * with contexts & known references 
   * alternative, earlier and original (i.e. by the fan translations above) translations. 
   * the official English phrases
   * => Reasoning: 
      * -> easier search-ability for phrases and words for future check ups
      * -> should prevent duplicates and wrong translations
      * -> matching the official English phrases with one or more translations
* Improving translations
* Moving some phrases to other topics
* Improving line breaks
* Incorporates the card-changing-commit b7e627186d5f29c3085e9da4498600c4d706e97d by @AnotherStranger

### CODE REVIEW

File: cah-fullformat-german.tex [hunk 1]
```diff
@@ -43,10 +43,10 @@
 \include{categories/eklig}
 \include{categories/fsk16}
 \include{categories/fsk18}
-\include{categories/politik}
+\include{categories/geschichte}
```
Comment (Vankog) [inline cah-fullformat-german.tex:46]: just sorted alphabetically

File: commands.tex [hunk 2]
```diff
@@ -138,7 +138,7 @@
 			\fi%
 		}%
 	%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-	% White Cards %
+	% Black Cards %
```
Comment (Vankog) [inline commands.tex:141]: fixing wrong comment

**Comment by Prior99** [comment]
Hi, what's the license of the official fan translations? Can we use them without mentioning?

**Comment by Vankog** [comment]
1. It is the same license as the main game. Mentioning must be done.
1. I mentioned them in the README.md for this reason
1. That's one reason why I opened #5 as well

File: categories/basis.tex [hunk 3]
```diff
@@ -21,276 +21,709 @@
...
-\whitecard{Basis}{Ackerbau.}%
 \whitecard{Basis}{Aktives Zuhören.}%
+% engl. "Active listening"
 \whitecard{Basis}{Alkoholismus.}%
-\whitecard{Basis}{Alten Leuten winken.}%
-\whitecard{Basis}{An Dingen lecken, um sie für sich zu beanspruchen.}%
-\whitecard{Basis}{Anfangen, zu singen und zu tanzen.}%
+% engl. "Alcoholism"
+\whitecard{Basis}{Alten Leuten zuzwinkern.}%
+% before: zuwinken
```
Comment (Prior99) [inline categories/basis.tex]: Remove the lines with `% before`. The new translations are much better and the git history should be enough if anybody cares about the old ones.

File: categories/basis.tex [hunk 4]
```diff
@@ -21,276 +21,709 @@
...
+\whitecard{Basis}{Charisma.}%
+\whitecard{Basis}{Dieses Ding, das deine Muskeln mit Elektroschocks aufbaut.}%
 \whitecard{Basis}{Dunkle und mysteriöse Mächte, die man nicht kontrollieren kann.}%
+% engl. "Dark and mysterious forces beyond our control"
 \whitecard{Basis}{Das Ende der Straße.}%
+% alternative to engl. "Road Head"
+\whitecard{Basis}{Das Gedränge.}%
+% alternative to engl. "The Hustle"
+\whitecard{Basis}{Das Glasdach.}%
+% original: Glasdecke
```
Comment (Prior99) [inline categories/basis.tex:69]: Remove the lines with `%original`. See comment above.

**Review by Prior99 — CHANGES_REQUESTED** [review]
Hey. Wow thanks for your great work, I can see that you put much effort into this. Please remove the lines as suggested by my comments and I'd be more than happy to merge this!

**Comment by Vankog** [comment]
I can see why you want to remove the "before" comments. I'll remove them.
However, I'd like to stand up for keeping the "original" ones. These are the ones from the fan translations by Ballz and Raphael. I introduced them to have a mapping. Otherwise, some of their phrases that I changed for one reason or another, might be reintroduced in the future. I'd like to prevent this from happening.
Especially, because they also added some German original phrases that don't show up in the engl. version.

**Comment by Prior99** [comment]
Sounds good!

**Comment by Vankog** [comment]
better?

**Comment by Prior99** [comment]
Yes. Accepted. Thanks for your contribution :)

**Pull request merged**